app/controllers/devise_token_auth/registrations_controller.rb

```diff
@@ -10,17 +10,10 @@ def create
       @resource.provider   = "email"
 
       # honor devise configuration for case_insensitive_keys
-      if resource_class.case_insensitive_keys.include?(:email)
-        @resource.email = sign_up_params[:email].try :downcase
-      else
-        @resource.email = sign_up_params[:email]
-      end
+      @resource.email.try :downcase! if resource_class.case_insensitive_keys.include?(:email)
```

Comment:
The `email` attribute will already be set if it was passed in as a parameter. We can just try and downcase it when required.